flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/WebMonitorEndpoint.java

```diff
@@ -495,6 +496,10 @@ protected List<Tuple2<RestHandlerSpecification, ChannelInboundHandler>> initiali
         final SubtaskMetricsHandler subtaskMetricsHandler =
                 new SubtaskMetricsHandler(leaderRetriever, timeout, responseHeaders, metricFetcher);
 
+        final SubtaskAttemptMetricsHandler subtaskAttemptMetricsHandler =
```

Comment:
I think this REST API is not in the scope of this FLIP. Therefore, I think we should remove it.